integration/js/script/run-test

```diff
@@ -115,6 +115,7 @@ case $1 in
       r video_test_processor.config.json
       #  TODO: Re-enable simulcast test after it has been fixed
       # r video_test_simulcast.config.json
+      r test_ap_video_test.config.json
```

Comment:
This wont work. Remember that all these video tests are using the meeting v2 demo while this one uses your new demo. You either need to break this out or try to relaunch the demo (not recommended).
Also there is an error in the GitHub test run (we need to identify why it did not fail the check).
```
java.io.FileNotFoundException: configs/test_ap_video_test.config.json (No such file or directory)
```